services/app-web/src/components/ChangeHistory/ChangeHistoryV2.test.tsx

```diff
@@ -0,0 +1,63 @@
+import { screen } from '@testing-library/react'
+import { renderWithProviders } from '../../testHelpers/jestHelpers'
+import { ChangeHistoryV2 as ChangeHistory } from './ChangeHistoryV2'
+import {
+    mockContractPackageSubmitted,
+    fetchCurrentUserMock,
+    mockValidStateUser,
+    mockValidCMSUser,
+    mockContractPackageUnlocked,
+} from '../../testHelpers/apolloMocks'
+
+describe('SubmissionTypeSummarySection', () => {
+    afterEach(() => {
+        jest.clearAllMocks()
+    })
+    const submittedContract = mockContractPackageSubmitted()
```

Comment:
why define this outside the test? In general it's best to call the mock functions every time you use one so that different tests don't smash each others' data